cluster/node-pools/master-default/userdata.clc.yaml

```diff
@@ -469,6 +470,14 @@ storage:
               - --pod-dns-ndots={{ .Cluster.ConfigItems.teapot_admission_controller_ndots }}
 {{- if eq .Cluster.ConfigItems.teapot_admission_controller_process_resources "true" }}
               - --pod-process-resources
+              - --pod-enable-env-inject
```

Comment:
This needs to use a separate config item because it's not related to resources at all.